client/boot/index.js

```diff
@@ -369,6 +369,10 @@ function reduxStoreReady( reduxStore ) {
 		require( 'lib/rubberband-scroll-disable' )( document.body );
 	}
 
+	if ( config.isEnabled( 'dev/test-helper' ) && document.querySelector( '.environment.is-tests ul.active-tests' ) ) {
+		require( 'lib/abtest/test-helper' )( document.querySelector( '.environment.is-tests ul.active-tests' ) );
```

Comment:
> Did you have trouble getting all the needed information from the jade index.js controller?

I just wanted to get fast prototype off the ground ASAP to stop wasting dev hours and get early feedback.
Yeah, I will probably follow with proper architecture and README :)